installer/manifests/keptn/charts/continuous-delivery/values.yaml

```diff
@@ -1,11 +1,23 @@
 openshift:
   enabled: false
 
-image:
-  registry: docker.io
-
 ingressConfig:
   ingress_hostname_suffix: "svc.cluster.local"
   ingress_protocol: "http"
   ingress_port: "80"
-  istio_gateway: "public-gateway.istio-system"
\ No newline at end of file
+  istio_gateway: "public-gateway.istio-system"
+
+gatekeeperService:
+  image:
+    repository: docker.io/keptn/gatekeeper-service
+    tag: latest
```

Comment:
In Keptn, all services in keptn/keptn are released together and, hence, have the same tag. Therefore, I would suggest to make the tag (and also the registry) configurable for all images at a single place in the values.yaml.

Reply:
Are you suggesting something like this?

values.yml
```
image:
  registry: docker.io
  tag: latest
```
and use this value in all deployment images?

one downside about this is when we have issue in like #2021 where we have no way of updating the image tag for a particular service. or this shouldnt be the case in the future?

Reply:
Great understanding! It makes totally sense what you proposed.